sql/mysqld.cc

```diff
@@ -5033,6 +5033,22 @@ a file name for --log-bin-index option", opt_binlog_index_name);
   }
   plugins_are_initialized= TRUE;  /* Don't separate from init function */
 
+#ifdef WITH_WSREP
+  LEX_CSTRING keyring_vault_name= {C_STRING_WITH_LEN("keyring_vault")};
```

Comment:
Not a big deal here but what about
`static const LEX_CSTRING keyring_vault_name= {C_STRING_WITH_LEN("keyring_vault")};`
?